ngraph/test/frontend/tensorflow/CMakeLists.txt

```diff
@@ -63,7 +63,6 @@ else()
 endif()
 
 add_dependencies(${TARGET_NAME} tensorflow_test_models)
-add_dependencies(${TARGET_NAME} tensorflow_ngraph_frontend)
```

Comment:
Was this dependency removed by mistake?

Reply:
no, the code above also has:
```cmake
target_link_libraries(${TARGET_NAME} PRIVATE frontend_shared_test_classes tensorflow_ngraph_frontend)
```
Which already sets a dependency between targets

Reply:
ok, got it, thanks